### PR TITLE
docs: Various edits, and addition of UI style guide

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

Various edits, and addition of UI text style guide to the Overseerr documentation.

### CODE REVIEW

File: docs/support/faq.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Perform these steps to verify the media item has a guid Overseerr can match.
 
 **A:** The logs are located at `<Overseeerr-install-directory>/logs/overseerr.log`
 
-## User Management
+## User management
```
Comment (sct) [inline docs/support/faq.md:71]: Why the inconsistency of capitalizing these titles? I feel like all capitals here looks better? Just like the main title? `Frequently Asked Questions`. This goes for the other similar changes as well

**Comment by sct** [comment]
@all-contributors please add @TheCatLady for docs

File: docs/using-overseerr/notifications/README.md [hunk 2]
```diff
@@ -24,6 +24,6 @@ Some agents may have specific configuration gotchas that will be covered in each
 Currently, you will **not receive notifications** for any auto-approved requests. However, you will still receive a notification when the media becomes available.
```
Comment (sct) [inline docs/using-overseerr/notifications/README.md:24]: Maybe we should update this as well.

File: docs/contributing/ui-text-style-guide.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# UI Text Style Guide
```
Comment (sct) [inline docs/contributing/ui-text-style-guide.md:1]: Let's just add this to the existing contribution guide? This doesn't need to be a part of the main docs.

File: docs/support/faq.md (hunk 1 above)
Comment (TheCatLady) [reply docs/support/faq.md:71]: I noticed that many of the other subtitles were not capitalized, so I changed it.  But I can just capitalize them all instead :)

**Review by sct — APPROVED** [review]

**Pull request merged**